packages/kotti-ui/source/kotti-navbar/KtNavbar.vue

```diff
@@ -72,32 +73,98 @@ export default defineComponent({
 		NavbarNotification,
 		NavbarQuickLink,
 	},
-	mixins: [clickaway],
 	props: makeProps(KottiNavbar.propsSchema),
 	setup(props: KottiNavbar.PropsInternal, { emit }) {
-		const mobileMenuToggle = ref(false)
+		const mobileMenuRef = ref<HTMLElement | null>(null)
+		const navbarRef = ref<HTMLElement | null>(null)
+		const tippyTriggerRef = ref<HTMLElement | null>(null)
+		const tippyInstanceRef = ref<Instance | null>(null)
 
 		provide(
-			IS_NAVBAR_NARROW,
-			computed(() => props.isNarrow),
+			KT_NAVBAR_CONTEXT,
+			computed(() => ({ isNarrow: props.isNarrow, theme: props.theme })),
 		)
+
+		useTippy(
+			navbarRef,
+			computed(() => ({
+				arrow: false,
+				appendTo: () => document.body,
+				content: mobileMenuRef.value ?? undefined,
+				interactive: true,
+				maxWidth: 'none',
+				offset: [0, 0],
+				onCreate(instance) {
+					tippyInstanceRef.value = instance
+				},
+				placement: 'bottom-start',
+				theme: props.theme
+					? `kt-navbar-${props.theme} kt-navbar-shadow`
+					: 'kt-navbar kt-navbar-shadow',
+				trigger: 'click',
+				triggerTarget: tippyTriggerRef.value,
+			})),
+		)
+
 		return {
 			classes: computed(() => ({
 				'kt-navbar--narrow': props.isNarrow,
 				[`kt-navbar--theme-${props.theme}`]: props.theme !== null,
 			})),
-			clickawayMobileMenu: () => (mobileMenuToggle.value = false),
-			mobileMenuToggle,
-			setIsNarrow: (isNarrow: boolean) => emit('setIsNarrow', isNarrow),
-			toggleMobileMenu() {
-				mobileMenuToggle.value = !mobileMenuToggle.value
+			hideMobileMenu: (event: MouseEvent) => {
+				emit('linkClick', event)
+				tippyInstanceRef.value?.hide()
 			},
+			mobileMenuRef,
+			navbarRef,
+			setIsNarrow: (isNarrow: boolean) => emit('setIsNarrow', isNarrow),
+			tippyTriggerRef,
+			Yoco,
 		}
 	},
 })
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
```

Comment:
but why not scope this?

if other usages of tippy in kotti rely on the theme, all the styles (bg-color, border-radius, color, etc) will leak, no?

Reply:
The theme is specific to the `kt-navbar`. It has to be unscoped because tippy attaches to body

Reply:
What Florian says. Scoping would hide all styles from this component, as the tippied content is not appended as a child here but at the html body.

Try it out yourself if you have to see it :-)